typescript/sdk/src/core/contracts.ts

```diff
@@ -16,71 +16,114 @@ import {
 } from '@abacus-network/core';
 import { types } from '@abacus-network/utils';
 
-import { AbacusAppContracts } from '../contracts';
-import { ChainName, ProxiedAddress } from '../types';
+import { IAbacusContracts } from '../contracts';
+import {
+  ChainName,
+  Connection,
+  ProxiedAddress,
+  RemoteChainMap,
+  Remotes,
+} from '../types';
+import { objMap } from '../utils';
+
+export type MailboxAddresses = ProxiedAddress & {
```

Comment:
It's not clear to me what `MailboxAddresses` represent in this case. I assume its that Inbox and Outbox addresses are `ProxiedAddress` + their validator manager. This is a nit, but in that case it might be nice to specify 

```
type InboxAddress = ProxiedAddress
type OutboxAddress = ProxiedAddress
type MailBoxAddresses = (InboxAddress | OutboxAddress) & { validatorManager: types.Address }
```